src/app/users/actions.ts

```
@@ -0,0 +1,62 @@
+"use server"
+
+import { prisma } from "@/lib/prisma"
+import { userSchema } from "./schema"
+import { z } from "zod"
+
+// Initialize Prisma Client (if not already done globally)
+// This is a common pattern to avoid multiple Prisma Client instances in Next.js server actions
+declare global {
+  var prisma: PrismaClient | undefined
```

Biome (2.1.2)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)